### ISSUE TITLE
Custom filed with type 'file' dont save filename.

### ISSUE THREAD

**Issue by Trysystems** [anchorcms/anchor-cms#1054]
### Summary

If load image in 'file' field, its upload by js, but info dont save to db. Other files dont uploads and dont saves to db. Array $_FILES is empty.

### Context details (if applicable)

Anchor 0.12.1 installed by composer(migration 211)

**Comment by daviddarnes** [comment]
Did you say you're uploading a file with the file extension called `.file` ?

**Comment by Trysystems** [comment]
No, file with .png or .zip. And this extensions was allowed in custom's field menu. 

So, in debugger on function files() (Extend.php) array $_FILES empty.

**Comment by daviddarnes** [comment]
Ah sorry, the title you gave confused me a little.

**Comment by Trysystems** [comment]
Cause - AJAX form submit script on editor.js line 195.  One else bug - this script has error in IE11 - its redirect page to 'admin/posts/edit/undefined' page.

**Comment by daviddarnes** [comment]
Ah I see, thanks for the additional info. Have you tried this in another browser? I realise this isn't a solid fix, but at least we can scope this down to a browser issue.

**Comment by Trysystems** [comment]
In last FF and Chrome there is no bug with redirection after ajax script end, but files still no downloads.
